src/controllers/trees.controller.ts

```diff
@@ -171,7 +171,7 @@ export class TreesController {
     try {
       let verifyCaptureProcessed;
       let domainEvent;
-      if (config.enableVerificationPublishing) {
+      if (config.enableVerificationPublishing === 'true') {
```

Comment:
I'd suggest using `config.enableVerificationPublishing?.toLowerCase()` here, to accommodate people setting the env var to `TRUE` or `True`.

An even better option would be to implement a simple, reusable env var parser in `config.js` to make `enableVerificationPublishing` a boolean and avoid the need for a string literal here.

Reply:
@nmcharlton , Updated the PR with your suggestions. Please take a look when you get a chance!